src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -182,6 +189,26 @@ def validate_step(self, step: str) -> bool:
                 return run_new_state is not None and run_new_state.result_state == RunResultState.SUCCESS
         return False
 
+    def _relay_logs(self, workflow, run_id):
+        for log in self._fetch_logs(workflow, run_id):
+            task_logger = logging.getLogger(log.component)
+            # we're relaying almost raw log events, so we need to call the logger directly
+            raw_log_event = task_logger._log  # pylint: disable=protected-access
+            raw_log_event(logging.getLevelName(log.level), log.message, ())
+
+    def _fetch_logs(self, workflow: str, run_id: str) -> Iterator[PartialLogRecord]:
```

Comment:
If we want to reuse this logic in the cli command it should be moved out of this class, see #1282

Reply:
Can be still here, see next PR 😉